docs/src/main/sphinx/connector/redshift.md

```diff
@@ -64,6 +64,32 @@ documentation](https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-configura
 ```{include} jdbc-authentication.fragment
 ```
 
+### UNLOAD configuration
+
+The connector brings parallelism by converting Trino query to corresponding 
+Redshift `UNLOAD`command to unload query results on to the S3 bucket in the 
+form of Parquet file and reading those Parquet files from S3 bucket. This 
+feature tend to bring query results faster compared to the default JDBC 
+approach when fetching large query result set from Redshift.
```

Comment:
The connector deletes the fragment location when it's done. It can fail to do this, so the docs should recommend setting up some lifecycle policies in the bucket to clean up orphaned files.

Reply:
Updated docs.